src/main/java/com/laytonsmith/annotations/EventIdentifier.java

```diff
@@ -0,0 +1,22 @@
+package com.laytonsmith.annotations;
+
+import com.laytonsmith.core.events.Driver;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Created by jb_aero on 3/22/2015.
+ */
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface EventIdentifier {
+
+	String className();
+
+	String priority();
```

Comment:
Because Sponge uses a completely different names and that seemed an acceptable way of matching the enum of either API, given only those of us adding new events will use it, and only when we add events that don't exist on other versions. The end user will never have to match something to that, and the value is only used on plugin load.